### ISSUE TITLE
There is a mistake in cfgs/shapenetpart/pointnext-s.yaml

### ISSUE THREAD

**Issue by zouyanmei** [guochengqian/PointNeXt#162]
The released version of cfgs/shapenetpart/pointnext-s.yaml is 
![image](https://github.com/guochengqian/PointNeXt/assets/45259702/2619878a-d06b-4af5-b1da-44b1b95f9495). 

But the code is 
![image](https://github.com/guochengqian/PointNeXt/assets/45259702/61033ce3-7431-462f-87ea-846ccb891040).

 I think this may be the main reason why we cannot reproduce the results.

**Comment by JacksonCampolattaro** [comment]
Hi @zouyanmei! With this issue patched, were you able to replicate the published results? I'm also having trouble.

**Comment by zouyanmei** [comment]
> Hi @zouyanmei! With this issue patched, were you able to replicate the published results? I'm also having trouble.

Yes, I achieved the published results with multiple GPUs. If you want to reproduce the results on ShapeNetPart, you must use multiple GPUs (at least 2 GPUs).

**Comment by JacksonCampolattaro** [comment]
> > Hi @zouyanmei! With this issue patched, were you able to replicate the published results? I'm also having trouble.
> 
> Yes, I achieved the published results with multiple GPUs. If you want to reproduce the results on ShapeNetPart, you must use multiple GPUs (at least 2 GPUs).

Thanks for the quick reply! I found #59 shortly after I messaged you, now I'm curious about the exact effect that's benefitting accuracy.

**Comment by guochengqian** [comment]
Fixed in the current config: \ now uses \, matching the released ShapeNetPart config. Closing as resolved in the current code/release.

**Comment by guochengqian** [comment]
Formatting correction to the previous closing comment: this is fixed in `cfgs/shapenetpart/pointnext-s.yaml`; the config now uses `global_feat: max,avg`.
